tur/shark/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/Bhaviktutorials/shark
+TERMUX_PKG_DESCRIPTION="🦈Future Of Phishing With less delay!!🦈"
+TERMUX_PKG_LICENSE="BSD-3-Clause"
+TERMUX_PKG_MAINTAINER="@termux-user-repository"
+TERMUX_PKG_VERSION="v2.5.1.0-beta"
```

Comment:
`TERMUX_PKG_VERSION` should starts with a number, so `v` should be removed.

Reply:
##### The source code of shark is release or named as *_v2.5.1.0-beta_* u can crosschk.